### ISSUE TITLE
`modelator apalache test <tla> <cfg>` should give a warning if no tests are found in the `<tla>` file.

### ISSUE THREAD

**Issue by danwt** [informalsystems/modelator#73]
Commit: _70e0fe1113e8107214c61dd6e5103da7cd5b5036_

It is possible for a user to pass a regular TLA spec and config to `apalache test`, meaning, one without _tests_ as generated by first running `generate-tests`. If an invariant is specified in the config then it will generate a counterexample. 

This is unlikely to be what the user is looking for, but there is no error message, so they won't realise.

**Comment by rnbguy** [comment]
For post-0.3.0 plans, I propose that we don't expose these subcommands to users.

Maybe we can have something similar to these,
```
modelator list-tests <tla> <cfg>
modelator test --checker apalache --type [json|tla] --testname <test> <tla> <cfg>
moderator parse <tla> 
```

**Comment by andrey-kuprianov** [comment]
yes, this sounds much better to me than the current CLI

**Comment by andrey-kuprianov** [comment]
@rnbguy I believe this is better to be addressed before 0.4.0 release as well

**Comment by rnbguy** [comment]
`modelator list Numbers.tla` should produce the following now.

```
{
  "status": "error",
  "result": {
    "NoTestFound": "Numbers"
  }
}
```
